src/SDKs/Search/DataPlane/Microsoft.Azure.Search/Fix-GeneratedCode.ps1

```diff
@@ -6,6 +6,12 @@ function Replace-InFile($filePath, $oldText, $newText)
 
 $generatedFolder = ".\GeneratedSearchIndex"
 
+# Delete any extra files generated for types that are shared between SearchServiceClient and SearchIndexClient.
```

Comment:
not sure if it was accidentally, just moved the commands from the old `generate.cmd` into this fixup script. Could well be that this is necessary if both Swaggers reference the same types...